internal/pkg/cli/interfaces.go

```diff
@@ -309,7 +309,7 @@ type envDescriber interface {
 }
 
 type pipelineStatusDescriber interface {
-	Describe() (*describe.PipelineStatusDesc, error)
+	Describe() (*describe.PipelineStatus, error)
 }
```

Comment:
So it looks like in your `describe` package, you have your `Describe()` method return a `HumanJSONStringer`, and in your `cli` package you are using the `describer` interface, so you should be able to get rid of the `pipelineStatusDescriber` interface entirely.